Dockerfile

```diff
@@ -43,6 +43,8 @@ ARG PYTHONOPTIMIZE=2
 ARG WKHTMLTOPDF_VERSION=0.12.4
 ARG WKHTMLTOPDF_CHECKSUM='049b2cdec9a8254f0ef8ac273afaf54f7e25459a273e27189591edc7d7cf29db'
 ENV OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
+    ADDON_INSTALL=false \
```

Comment:
The WAIT_CONTAINER stuff is OK, but drop this `ADDON_INSTALL` feature. It makes the image more complex, and after all you can do the same replacing your:

```yaml
installer:
  environment:
    ADDON_INSTALL: "true"
```

by:

```yaml
installer:
  command: install-addons
```

... so no real benefit, unless I'm missing something... 🤔